showcase/src/app/news/components/news-item/news-item.component.scss

```diff
@@ -0,0 +1,14 @@
+.mat-content-image {
```

Comment:
Now, the name of all SCSS files should have an underscore "_" at the beginning ;)